### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by Luffytaro22**
> # Add, Remove and Modify functionalities 📃 
> gif

> ## Features 🆕 
> * `Add` a `task` when the user press `enter` 👇 
> * `Remove` a `task` when the user press the `trash icon` 🗑️ 
> * `Modify` the task in the `local storage` 💾 
> * When a task is `removed`, the `index` property in the array `changes` ✔️

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>To Do List</title>
-  <link rel="icon" href="../lista-de-deseos.png"> <!-- Web Icon -->
+  <link rel="icon" href="C:\Users\manue\Documents\To-do-list\src\assets\lista-de-deseos.png"> <!-- Web Icon -->
   <script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script> <!-- Font Awesome -->
 <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
 <body>
```
Comment (Eternalgratis) [inline dist/index.html:11]: - I can see that you have implemented some of the project requirements but the user interface isn't showing any of them. The add task, delete task, edit task, and clear all completed button isn't working. Kindly check if you are linking your javascript files correctly to your HTML and please implement the changes.

File: src/styles.css [hunk 2]
```diff
@@ -18,9 +18,25 @@ h1 {
...
+  border: none;
+  padding-top: 15px;
+  background: none;
+}
+
+.background {
+  background-color: blanchedalmond;
 }
 
 #list-container {
```
Comment (Eternalgratis) [inline src/styles.css:42]: [Optional changes]

- Please consider removing the bullets on the input and on the subtitle of the Todolist as they aren't necessary in this case

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
Hi @Luffytaro22 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there

## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ Linter error checks validated


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Luffytaro22, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**